app/services/api/v3/participant_declarations_query.rb

```diff
@@ -118,15 +119,26 @@ def delivery_partner_ids
       end
 
       def with_joins(scope)
-        scope.left_outer_joins(
-          participant_profile: [
-            [schedule: :cohort],
-            { induction_records: [
-              :cohort,
-              { induction_programme: :partnership },
-            ] },
-          ],
-        )
+        # Get the latest induction record for each participant declaration (kind of - may need more thought)
+        subquery = InductionRecord
```

Comment:
@cwrw to have a think about this

Reply:
chatted in person: from @ethax-ross best thing is to stamp the cohort on the declaration on creation to avoid all this. We will need to think about backfilling submitted/ineligible declarations